github/repos.go

```diff
@@ -318,13 +318,13 @@ func (s *RepositoriesService) GetCodeOfConduct(ctx context.Context, owner, repo
 	// TODO: remove custom Accept header when this API fully launches.
 	req.Header.Set("Accept", mediaTypeCodesOfConductPreview)
 
-	conduct := new(Conduct)
-	resp, err := s.client.Do(ctx, req, conduct)
+	codeOfconduct := new(CodeOfConduct)
```

Comment:
`codeOfconduct` has lowercase second "c", instead of `codeOfConduct`. But you can use a shorter variable name here too.